README.md

```diff
@@ -92,6 +93,8 @@ prometheusEndpointParams:
 # under "MQ PCF API specific statistics" section.   
 PCFParameters:
 # Collect additional metrics? If false, all settings in this section below are ignored. 
+# If yes, additional metrics will be collected for all queues, channels and listeners listed below. 
+# Note: User must have permission to inquire  channels, queues and listeners listed below! 
```

Comment:
This line should be in a "user" section, where all permissions are noted.
Otherwise, it can be mislead.